exoplayer-flac/src/main/.gitiginore

```diff
@@ -1 +0,0 @@
-libs/
```

Comment:
The file was named incorrectly and the correct file is right beside it. It had an 'i' too much